main/tests.py

```diff
@@ -77,3 +78,34 @@ def test_dog_owner_homepage_is_visible_for_dog_owner(self, client, create_dog_ow
         response = client.get("/homepage/")
         assert response.status_code == 200
         assert list(response.context['daycares']) == list(DayCare.objects.all())
+
+
+@pytest.mark.django_db
+class TestOrdersView:
+    def test_orders_page_is_visible_for_dog_owner_user(self, client, create_dog_owner_user):
+        client.force_login(user=create_dog_owner_user.user)
+        response = client.get("/orders/")
+        assert response.status_code == 200
+
+    def test_orders_page_is_visible_for_daycare_user(self, client, create_daycare_user):
+        client.force_login(user=create_daycare_user.user)
+        response = client.get("/orders/")
+        assert response.status_code == 200
+
+    def test_only_relevant_orders_are_displayed_for_daycare(self, client, create_daycare_user):
```

Comment:
it doesn't test only relevant orders are displayed. it tests that ALL the orders are displayed...

Reply:
Changed the name of the method to "test_only_orders_which_belongs_to_user_as_daycare_are_displayed". Same goes for dog owner.